### PR TITLE
Align dates by left in the load file dialog

### PR DESCRIPTION
**Pull request by kant2002**
See #2501

### CODE REVIEW

**Comment by ihhub** [comment]
Hi @kant2002 , I converted this pull request to draft since we're still in the middle of discussion about the way to fix the issue.

**Comment by kant2002** [comment]
@ihhub consensus is reached. Please take a look.

File: src/fheroes2/dialog/dialog_selectfile.cpp [hunk 1]
```diff
@@ -75,10 +75,16 @@ class FileInfoListBox : public Interface::ListBox<Maps::FileInfo>
 void FileInfoListBox::RedrawItem( const Maps::FileInfo & info, s32 dstx, s32 dsty, bool current )
 {
     char short_date[20];
+    char short_hours[20];
```
Comment (ihhub) [inline src/fheroes2/dialog/dialog_selectfile.cpp:78]: Even the old variables are written in a different way could you please name them using camelCase style without _ in the middle?

Comment (kant2002) [reply src/fheroes2/dialog/dialog_selectfile.cpp]: You would not ask me twice to do that.

Comment (ihhub) [reply src/fheroes2/dialog/dialog_selectfile.cpp]: Well done!

**Comment by ihhub** [comment]
@kant2002 , thank you so much for improving the code of the project!

**Pull request merged**